python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -905,10 +900,24 @@ def get_event_records(
                         "Could not resolve event record as EventLogEntry for id `%s`.", row_id
                     )
                     continue
-                else:
-                    event_records.append(
-                        EventLogRecord(storage_id=row_id, event_log_entry=event_record)
-                    )
+
+                if event_records_filter.tags and not self.has_table(AssetEventTagsTable.name):
```

Comment:
is the indentation off on this block?

Reply:
Mind elaborating? I don't think so, but maybe I wrote it in a more confusing way than I needed to?

Reply:
oh, you're right... didn't see this was part of a different function.

Reply:
I think you still need to add a check in line 867:

```
        if event_records_filter.tags and not self.supports_intersect and self.has_table(AssetEventTagsTable.name):
```

Reply:
You are right. Thanks for catching.